CHANGELOG.md

```diff
@@ -23,15 +23,11 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
```

Comment:
changelog files are not meant to be changed manually!

Reply:
On it. I will revert these.